### ISSUE TITLE
whichbrowser.net

### ISSUE THREAD

**Issue by jhaag75** [WebPlatformTest/HTML5test#529]
html5test.com appears to not be working on the home page.

Perhaps the dependency on whichbrowser.net is the culprit? That site is also not working from firefox or chrome.

**Comment by qwertimus** [comment]
Same problem for me; html5test.com does not load the actual test, and whichbrowser.net appears to be down.  
Safari (12604.4.7.1.6)

**Comment by anthonyxwatson** [comment]
Yup. Hangs in mobile site and desktop. From desktop, it looks like the call to whichbrowser.com is stuck as site is down.

**Comment by qwertimus** [comment]
Looks like both sites are working properly again.  
It might be worthwhile looking at ways to limit the impact of whichbrowser.net downtime.

**Comment by brynne8** [comment]
Same. Please run a browser check instead of depending on whichbrowser.net
